programs/drift/src/state/state.rs

```diff
@@ -54,6 +54,7 @@ pub enum ExchangeStatus {
     LiqPaused = 0b00010000,
     FundingPaused = 0b00100000,
     SettlePnlPaused = 0b01000000,
+    AmmImmediateFillPaused = 0b10000000,
```

Comment:
do we want this at per market level? or other safety checks handle?

Reply:
yeah, adding per market flip + review of `amm.can_skip_auction_duration` criteria would be great